components/camel-elasticsearch-rest-client/src/main/java/org/apache/camel/component/elasticsearch/rest/client/ElasticsearchRestClientComponent.java

```diff
@@ -19,18 +19,162 @@
 import java.util.Map;
 
 import org.apache.camel.Endpoint;
+import org.apache.camel.spi.Metadata;
 import org.apache.camel.spi.annotations.Component;
 import org.apache.camel.support.DefaultComponent;
+import org.elasticsearch.client.RestClient;
 
 @Component("elasticsearch-rest-client")
 public class ElasticsearchRestClientComponent extends DefaultComponent {
 
+    @Metadata(autowired = true)
```

Comment:
label = advanced

Reply:
Thanks for catching it. It's updated now.